src/python/bot/fuzzers/honggfuzz/engine.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""honggfuzz engine interface."""
+
+import glob
+import os
+import re
+
+from bot.fuzzers import dictionary_manager
+from bot.fuzzers import engine
+from system import environment
+from system import new_process
+
+_DEFAULT_ARGUMENTS = [
```

Comment:
can move this and values 2048, 25 into its own vars in constants.py

Reply:
I don't think we need a constants.py here :) I think it's better to have vars closer to where they're used for readability, unless they're shared. In this case this file should be quite self contained. Happy to change this if we start needing these vars elsewhere or if you feel strongly.

(moved the timeout and rss limit into their own vars).

Reply:
Ok, here is fine.